### ISSUE TITLE
Issues running tests

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/infrastructure-integration-test#1]
Tried to run the tests on my laptop and ran into a couple issues:

1. Was unable to the curl to get geckodriver (got a redirect issue) - so I just downloaded latest manually from https://github.com/mozilla/geckodriver/releases/

2. After running the tests, it didn't wait for the duo push before continuing for me, so the test failed

**Comment by jcoyne** [comment]
Thanks for testing this @peetucket.  Do you think you took longer than 100 seconds to get the push notification returned? That's what I have it set to right now: https://github.com/jcoyne/infrastructure-integration-test/blob/master/spec/features/preassembly_spec.rb#L33. Or are you saying it failed right away?

**Comment by peetucket** [comment]
It was faster than I could respond to the duo request - it failed right away. But that seems odd given your timeout. Let me try again.

**Comment by peetucket** [comment]
now works for me
